elastic/kibana
# [AO] Add alert time range annotation for metric threshold

Resolves #153851

## Summary

This PR adds alert time range annotation for the metric threshold rule details page.

![image](https://user-images.githubusercontent.com/12370520/230080136-434bb586-c00e-4f36-aa11-525a38f8a650.png)
**Note**
I changed the rule criteria to stop the alert, hence the weird graph!

## 🧪 How to test
1. Add `xpack.observability.unsafe.alertDetails.metrics.enabled: true` to the Kibana config
2. Generate a metric threshold alert
3. Go to the related alert details page and check the annotation
